lib/ex_unit/extra.ex

```diff
@@ -25,4 +25,38 @@ defmodule ExUnit.Extra do
     end
   end
 
+
+  @doc """
+  Compares two MapSets and asserts that they are equal.
```

Comment:
Backticks on `MapSet`?

Reply:
Also, this doesn't quite accurately depict the use case. This accepts any two enumerables, and then treats them as we sets for comparison sake.

Suggest: "Convert two `Enumerable`s into `MapSet`s and assert that they are equal"